courses/free-courses-en.md

```diff
@@ -1188,6 +1188,7 @@
 #### React Native
 
 * [CS50's Mobile App Development with React Native](https://www.edx.org/course/cs50s-mobile-app-development-with-react-native) - edX
+* [FREE React Native Bootcamp in 30 HOURS](https://youtu.be/q3HE1dFiJBI?si=1ABf5i8ARyR5m_Me) - Codesistency(Youtube)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes